payjoin-cli/src/app/v1.rs

```diff
@@ -104,7 +104,9 @@ impl App {
         let pj_part = payjoin::Url::parse(pj_part)
             .map_err(|e| anyhow!("Failed to parse pj_endpoint: {}", e))?;
 
-        let pj_uri = PjUriBuilder::new(pj_receiver_address, pj_part).amount(amount).build();
+        // None is needed for for the OhttpKeys because we fetch `payjoin-io` which fetch `v2`
+        // feature from `payjoin` crate by default.
+        let pj_uri = PjUriBuilder::new(pj_receiver_address, pj_part, None).amount(amount).build();
```

Comment:
Are you sure? This `v1` module should only compile withOUT payjoin v2 feature since the signature of that function looks like this:

```
    pub fn new(
        address: Address,
        pj: Url,
        #[cfg(feature = "v2")] ohttp_keys: Option<OhttpKeys>,
    ) -> Self {
```

The inclusion of `payjoin-io` for all feature configurations of `payjoin-cli` must have forced `payjoin/v2` even for `payjoin-cli/not(v2)` and caused a compile time error here. Making `payjoin-io` an explicit dependency of `payjoin-cli/v2` will render this change unnecessary.